clcache.py

```diff
@@ -925,6 +925,19 @@ def resetStatistics(cache):
     stats.save()
     print('Statistics reset')
 
+def cleanCache(cache):
+    cfg = getConfiguration()
+    stats = CacheStatistics(cache)
+    cache.clean(stats, cfg.maximumCacheSize())
+    stats.save()
+    print('Cleaning cache')
```

Comment:
I think these print statements should be shown *before* doing the action (e.g. cleaning the cache).